### PR TITLE
add html and css pages

### PR DESCRIPTION
**Pull request by Meazajao**
I added the rest of html and css filles

### CODE REVIEW

File: images/Birds/Cassowary1.png
Comment (bushra-rauf) [inline images/Birds/Cassowary1.png:1]: hi can you

Comment (bushra-rauf) [reply images/Birds/Cassowary1.png:1]: The all images has different sizes. That always has an issue if they are not the same sizes. The image will stretch. Please find the same sizes pictures 2000pixel is best.

File: mammals.html
Comment (bushra-rauf) [inline mammals.html:1]: all good

File: reptiles.html
Comment (bushra-rauf) [inline reptiles.html:1]: 👍

**Review by bushra-rauf — CHANGES_REQUESTED** [review]

**Review by bushra-rauf — CHANGES_REQUESTED** [review]
images need to change

File: images/Birds/Cassowary1.png
Comment (Meazajao) [reply images/Birds/Cassowary1.png:1]: We can easily fix that with CSS! By using properties like object-fit: cover or setting a consistent width and height for the images, we can ensure they appear uniform without stretching or distorting.:)

Comment (Souad-Sud) [reply images/Birds/Cassowary1.png:1]: I resized the image so it good

**Review by Souad-Sud — APPROVED** [review]

**Review by Souad-Sud — APPROVED** [review]

**Review by bushra-rauf — APPROVED** [review]
all checks!

**Pull request merged**